folder/main.py

```diff
@@ -112,13 +158,31 @@ def __init__(self, conf):
         logging.info("Watching " + self.path + " for " + ",".join(patterns))
 
     def on_created(self, event):
+        # Skip index.json if it exists
+
+        if os.path.basename(event.src_path) == "index.json":
+            logging.info(f"Skipping index.json file")
+            return
+
         logging.info(f"Starting Processing of file {event.src_path}")
         sha256asset = common.sha256sum(event.src_path)
 
+        lock_file = "" 
+        if "lockFile" in self.config:
+            lock_file = self.config["lockFile"]
```

Comment:
This field is not mentioned in the docs.

Reply:
I think it does?
```
- lockFile: Wait for file not to exist before processing file
```